### DISCUSSION TITLE
NCICT 3.0.20240125 does not support Windows 7

### DISCUSSION THREAD

**Discussion by kutsen** [ncidose/ncidosetools#27] [NCICT]
Trying to run NCICT 3.0.20240125 under Windows 7 Ultimate SP1 (x64). Got an error "The procedure entrypoint CreateFile2 could not be located in the kernel32.dll." The following error is "Failed to load plugin GZipx64.dll" "The specified procedure was not found". NCICT version 3.0.20230428 runs OK on the same machine.

**Comment by choonsiklee** [comment]
We will check and get back to you shortly.

**Reply by kutsen** [reply]
The same issue with NCICT version 3.0.20240229

**Reply by choonsiklee** [reply]
Unfortunately, we found that Windows 7 is not supported by Xojo, the language used for all NCI dose programs since its release 2022r3. Two options: we upload the legacy versions for some Windows 7 users or you upgrade your Windows to 10 or later versions. Please let us know which option you prefer.

**Reply by kutsen** [reply]
I think that keeping the legacy version is the best.

**Reply by choonsiklee** [reply]
The legacy versions are now available. Please give them a try!
